pkg/reconciler/v1alpha1/broker/broker.go

```diff
@@ -335,14 +337,14 @@ func (r *reconciler) reconcileChannel(ctx context.Context, get func() (*v1alpha1
 	// TODO Determine if we want to update spec (maybe just args?).
 	// Update Channel if it has changed. Note that we need to both ignore the real Channel's
 	// subscribable section and if we need to update the real Channel, retain it.
-	//expected.Spec.Subscribable = c.Spec.Subscribable
-	//if !equality.Semantic.DeepDerivative(expected.Spec, c.Spec) {
+	// expected.Spec.Subscribable = c.Spec.Subscribable
+	// if !equality.Semantic.DeepDerivative(expected.Spec, c.Spec) {
```

Comment:
I don't understand why this is commented.

Reply:
This is commented based on the comment on line 337: `TODO Determine if we want to update spec (maybe just args?).` If we decide to update, then this code will do so. 

Would you prefer I remove the code and just have the bare TODO?

Reply:
I wasn't sure how recent this commented code was, or how easy it would be to replicate.

Reply:
Removed.